include/RAJA/util/View.hpp

```diff
@@ -296,6 +297,76 @@ RAJA_INLINE AtomicViewWrapper<ViewType, AtomicPolicy> make_atomic_view(
   return RAJA::AtomicViewWrapper<ViewType, AtomicPolicy>(view);
 }
 
+struct layout_left{};
+struct layout_right{};
+
+template<typename LAYOUT>
+struct Reshape;
+
+template<typename LAYOUT>
+struct Reshape;
+
+template<std::size_t Id, std::size_t...Ids>
+constexpr auto get_first_index()
+{
+  return Id;
+}
+
+template<std::size_t...Is>
+struct Reshape<std::index_sequence<Is...>>
+{
+  template<typename T, typename...Ts>
+  static auto get(T *ptr, Ts... s)
+  {
+    constexpr int N = sizeof...(Ts);
+    std::array<RAJA::idx_t, N> extent{s...};
+
+    auto custom_layout =
+      RAJA::make_permuted_layout(extent, std::array<RAJA::idx_t, N>{Is...});
+
+    constexpr auto unit_stride = get_first_index<Is...>();
+
+    return RAJA::View<T, RAJA::Layout<N, RAJA::Index_type, unit_stride>>
```

Comment:
Bug found here! unit_stride should be the last value of the prameter pack -- TODO: FIX

Reply:
Now fixed.